pages/documentation/developer/tutorials/diagram/nad-svg-writing.md

```diff
@@ -19,7 +19,7 @@ First of all, we need some Maven dependencies.
 <dependency>
     <groupId>com.powsybl</groupId>
     <artifactId>powsybl-starter</artifactId>
-    <version>2022.0.0</version>
+    <version>2023.0.0</version>
```

Comment:
Below you have used the 2023.0.1

Reply:
True, I forgot to update it, thanks for pointing that out!